styleguide/progress.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+@import 'typography';
+
+// progress bars
+.kiln-progress-wrapper {
+  bottom: 100%;
+  left: 0;
+  pointer-events: none;
+  position: absolute;
+  width: 100%;
+
+  .bar {
+    bottom: 0;
+    height: 2px;
+    left: 0;
+    position: absolute;
+    width: 100%;
+  }
+}
+
+// status messages
+.kiln-status {
+  @include primary-text();
+
+  bottom: 100%;
```

Comment:
In most circumstances, yes. In this one, it's overlaying other elements on the page (and it doesn't care)